configs/nri-prometheus-config.yml.sample

```diff
@@ -26,10 +26,10 @@ integrations:
       # It does not include verbose mode. This can lead to a high log volume, use with care.
       audit: false
 
-      # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
+      # The HTTP client timeout when fetching data from endpoints. Defaults to "5s" if it is not set.
       # scrape_timeout: "30s"
```

Comment:
since this is commented we can reflect the real value 5s

Reply:
Changed here: [7c04876](https://github.com/newrelic/nri-prometheus/pull/381/commits/7c048766863e62e028c94944db44d7123ec9f015)